src/index.tsx

```diff
@@ -215,20 +225,29 @@ export class App extends React.Component<{}, AppState> {
       event.preventDefault();
       // Select all: Cmd-A
     } else if (event[META_KEY] && event.code === "KeyA") {
-      elements.forEach(element => {
+      let newElements = [...elements];
+      newElements.forEach(element => {
         element.isSelected = true;
```

Comment:
One mutation left here

Reply:
Not a mutation of the `elements`, bu of the members. So we're good here.